src/client.js

```diff
@@ -280,14 +435,21 @@ class TouchPortalClient extends EventEmitter {
     });
   }
 
-  connect(options = {}) {
```

Comment:
This is going to break every current usage of the API, isn't it?

IMHO the option object is more flexible, eg. the next PR adds some options.

But if you really want to accept parameters directly like this, you need to at least check that the first parameter is _not_ an object type.  And keep the old code which handles it being an object. This will preserve backwards compatibility (BC) and keep flexibility.

Reply:
Ah yeah. Forgot about that.